services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/modelsTable.tsx

```diff
@@ -15,6 +19,37 @@ const ModelsTable = ({ activityId, prompt }) => {
     queryFn: fetchModels
   });
 
+  const updateModelNote = (id, notes)  => {
+    request.put({
+      url: `${process.env.DEFAULT_URL}/api/v1/comprehension/automl_models/${id}`,
+      json: { automl_model: {notes} }, },
+      (error, httpStatus, body) => {
+        if (error) {
```

Comment:
Error handling was not specified in the spec. Do we have a default input error handler that I can use?

Reply:
so we actually already have all of our model API util functions [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Staff/utils/comprehension/modelAPIs.ts). You'd want to just build an update one similar to the create. However, we are in the process of updating all these utils to have better error handling. You can see an example here with the [Rule API util functions](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Staff/utils/comprehension/ruleAPIs.ts#L69), so it would be ideal to match this error handling

Reply:
I used the commented out code stubs, thanks for surfacing that.